### ISSUE TITLE
Set focus to card-body after clicking Edit/Undo

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#238]
Currently, upon clicking `Edit this section`/`Undo your confirmation` buttons we are setting focus to the next-focusable (interactive) element. If there is no interactive element in the section's card-body then focus is set to the `Done`/`Confirm` button, skipping over the card-body content. SODA reports that this is unintuitive, and instead focus should shift to the newly revealed content (i.e., the card-body).

Here's how we might achieve this:
- add `tabindex="-1"` to each section's card-body. (This allows the card-body div to receive focus, without adding it to the sequential keyboard Tab order.)
- upon clicking `Edit this section`/`Undo your confirmation`, set focus to that section's div `id="step-X-body"`.

Notes:
- I'm not able to fully test this approach by modifying attributes within the browser, so it may take some trial-and-error.
- Currently, upon clicking `Done`/`Confirm` focus is set to `Edit this section`/`Undo your confirmation` button. This is appropriate and does not need to be changed.

### Reported by SODA
> 
> [Summary]
> Keyboard focus is lost or misplaced due to user interaction or content update
> 
> [Description]
> Keyboard focus is lost or placed on the wrong element during user interaction, content refresh or update, or other reason.
> When activating the "Undo your confirmation" the focus shift to the "confirm" button at the bottom of the section. Focus should remain on the control or shift to the newly revealed content. 
> 
> [Impact]
> Critical
> 
> [Recommended to fix]
> Rule : 
> The focus MUST be purposely moved or set (via JavaScript) onto the appropriate element when the user's action requires a change of context or location for effective keyboard or touch interaction.
> 
> How To Fix: 
> Fix this issue by explicitly placing focus on a logical element when content is removed, refreshed, or added, for example:
> 1. For content added to the screen in reaction to a user-fired event, focus should be shifted to the new content - such as in single page applications.
> 2. For content removed from the screen in reaction to a user-fired event, focus should be shifted to the next logical place in the interaction.
> 
> Reference: 
> Deque University: https://dequeuniversity.com/class/input-methods/keyboard-input/focus-management
> Deque University:https://dequeuniversity.com/class/dynamic-updates/notify-users/move-focus
> Deque University: https://dequeuniversity.com/class/dynamic-updates/ajax/single-page-applications
> 
> Background: 
> One of the biggest challenges when creating rich web interfaces using JavaScript is the management of focus when new content or controls are added or removed from the page. The presentation or modification of content on the screen may require that the user interact with -or at the very least, take notice of - the new or changed content. It is important to have a clear indication of page content that has been updated. This allows the user to understand the change and also allows a keyboard user to interact with any new content.
